### ISSUE TITLE
[Request] Be more visual when only start point be set.

### ISSUE THREAD

**Issue by kaikaiiiiiii** [mifi/lossless-cut#93]
When both start end points are set, there will be a darker block between it. It's good.

But when only one of start/end points set, there's nothing showed on UI, and some times I forget whether I had set it or not. If there are some changes it will be better.

For example, a red triangle on top of the progress bar, or a green line where start/end point be set.

I think green line should be easier.

**Comment by mifi** [comment]
Afaik it's not possible to set only one of start of end point. Do you mean setting start point BEFORE end point? That we could prevent

**Comment by reefhound** [comment]
You both have me confused. You can certainly set only a Cut Start or only an Cut End. And it visually shows the darker area from the Start Cut to the end of movie, or from the beginning of movie to the End Cut, just like it does when aboth a Start and End are set. I do this all the time.

That said, it would be more apparent to show it by changing primary colors than by changing shades of gray. If you do implement that other suggestion about lossless snip, perhaps Red for what is to be deleted and Green for what is to be kept will make it very obvious to the user.

**Comment by kaikaiiiiiii** [comment]
I'm sorry it's my fault. I don't describe the problem clear enough. Let me try again.

First, this is a normal status of cutting video: darker part shows the segments preparing to cut, the red line shows where now time is.
![losslesscut_2018-10-04_17-26-53](https://user-images.githubusercontent.com/1635138/46465272-cb380e00-c7fa-11e8-9481-5b89be993f07.png)

Then at this time I press "I". It set now time as new start, clear the darker part, as purpose. After this operation, there're still a start point and before it an end point, which, don't show any thing on UI.
![losslesscut_2018-10-04_17-26-53](https://user-images.githubusercontent.com/1635138/46465395-118d6d00-c7fb-11e8-8b3f-a9b70d7f7fa7.png)

Then I move on to seek a new end point. Unfortunately, for some reason, in some case, by some incident, I spend a lot time to make it done. So I forget whether I set the new start point or not.
![losslesscut_2018-10-04_17-32-10](https://user-images.githubusercontent.com/1635138/46465536-71841380-c7fb-11e8-97b1-7261635d93de.png)
You can blame my low memory brain, but it could be a new feature.

If there're always some indicators on progress bar, it would be some more convenient. I painted on the shot. ( Green line as start point, blue line as end point, and red line as now time point).
![losslesscut_2018-10-04_17-32-10](https://user-images.githubusercontent.com/1635138/46466064-d9872980-c7fc-11e8-84b3-ba0d4a8db71c.png)

**Comment by reefhound** [comment]
I get it. When you select a new start point that is later than the end point, the range highlighting goes away because that is an invalid range, leaving no visual indication (other than the cut numbers) of the new start point you selected. I'm guessing that is an Electron control and not sure how much it can be configured. Your last screenshot would be confusing to me, trying to remember which colors mean what. 

An easy solution would be to automatically set the end point as end of movie when a new start point is selected that isafter the previous end point. Conversely, automatically set start point as start of movie when a new end point is selected that is before the previous start point. That would be simple math and provide a visual range.

**Comment by mifi** [comment]
This has been improved in the newest version 2.0.0

**Comment by mifi** [comment]
Please reopen if still confusing
